src/utils/expenditures.ts

```diff
@@ -0,0 +1,4 @@
+export const getExpenditureDatabaseId = (
+  colonyAddress: string,
+  nativeExpenditureId: number,
+): string => `${colonyAddress}_${nativeExpenditureId}`;
```

Comment:
Maybe checksum the colony address in here, to avoid having to do it in the handlers